TgeeT/Questioner
# feature #163046677 down vote question endpoint

#### What does this PR do?

Create an endpoint for meetups and allow a user to down vote a question
#### Description of Task to be completed?

write failing test, 
write the bear minimum code to pass the down vote tests

#### How should this be manually tested?

follow the steps below:
1. git clone https://github.com/tgeet/Questioner.git
2. git checkout to ft-downvote-question-163046677 branch
3. flask run the app and use postman to verify

### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163046677
